internal/model/manifest.go

```diff
@@ -18,36 +17,72 @@ func (m ManifestName) String() string { return string(m) }
 // NOTE: If you modify Manifest, make sure to modify `Manifest.Equal` appropriately
 type Manifest struct {
 	// Properties for all manifests.
-	Name         ManifestName
-	tiltFilename string
+	Name          ManifestName
+	tiltFilename  string
+	dockerignores []Dockerignore
+	repos         []LocalGithubRepo
 
-	// TODO(maia): buildInfo
+	// Info needed to build an image. Can be StaticBuild, FastBuild... etc.
+	buildInfo buildInfo
```

Comment:
is there some reason why this isn't a DockerInfo? afaik, there are no other types of buildInfos. it seems like a meaningless interface

Reply:
Fair. Theoretically in future we might want to use different image builders, but i agree that we don't need that flexibility built in right now.

Reply:
fixed!